html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <div>Email
+      <input type="text" name="email"/>
+    </div>
+    <div>Why is coding cool?</div>
+    <select name="select an option">
+      <option value="select an option">Select an option</option>
+      <option value="becasue it's fun">Because it's fun</option>
+      <option value="because it's interesting">Becasue it's interesting</option>
+      <option value="becasue it's powerful">Becasue it's powerful</option>
+    </select>
+    <div>Favorite Language (choose one)
+      <br/>
+      <input type="radio" name="language" value="HTML"/> HTML
+      <input type="radio" name="language" value="CSS"/> CSS
+      <input type="radio" name="language" value="JavaScript"/> JavaScript
+    </div> Would you like to sign up for more information
+      <br/>
+      <input type="checkbox" name="information" value="Sign me up!"/> Sign me up!
+    <div> Comments
```

Comment:
This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

Also, the `textarea` should be a sibling to the `label`, not a child of it.